src/btree/bt_compact.c

```diff
@@ -384,7 +384,8 @@ __wt_compact(WT_SESSION_IMPL *session)
          * Compact pulls pages into cache during the walk without checking whether the cache is
          * full. Check now to throttle compact to match eviction speed.
          */
-        WT_ERR(__wt_evict_app_assist_worker_check(session, false, false, &eviction_happened));
+        WT_ERR(
+          __wt_evict_app_assist_worker_check(session, false, false, false, &eviction_happened));
```

Comment:
even compact is mandatory, just curious.

Reply:
The flag is "re-worded", it's "not interruptible by user". But regardless of the flag's value, it wouldn't be since it's an internal session.